### PR TITLE
Pointless norm on vector fibers

### PR DESCRIPTION
**Pull request by mateuszbaran**
Implements the feature discussed here: https://github.com/JuliaManifolds/LieGroups.jl/pull/8#discussion_r1807766685 .

### CODE REVIEW

**Comment by mateuszbaran** [comment]
Oh, I'm not sure if we can have that:
```julia
norm(itr, p::Real) @ LinearAlgebra /opt/hostedtoolcache/julia/1.10.5/x64/share/julia/stdlib/v1.10/LinearAlgebra/src/generic.jl:595
norm(M::ManifoldsBase.VectorSpaceFiber{𝔽} where 𝔽, X) @ ManifoldsBase ~/work/ManifoldsBase.jl/ManifoldsBase.jl/src/VectorFiber.jl:11
```
Especially after #206 .

**Comment by mateuszbaran** [comment]
My local testing indicates that it doesn't break anything in Manifolds.jl.

File: NEWS.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.15.20] 24/10/2024
+
+### Changed
+
+* `norm` function on `VectorSpaceFiber` (such as `TangentSpace`) now needs to be called without the point.
```
Comment (kellertuer) [inline NEWS.md:12]:
```suggestion
* `norm` function on `VectorSpaceFiber` (such as `TangentSpace`) now needs to be called without the point. The passed point was already ignored before.
```

**Review by kellertuer — APPROVED** [review]
± the small improvement that this is a bugfix actually, this all looks good.

**Pull request merged**